python_modules/dagster/dagster/core/definitions/partition.py

```diff
@@ -770,6 +770,13 @@ def tags_for_partition_fn(self) -> Optional[Callable[[Partition[T]], Dict[str, s
     def get_partition_keys(self, current_time: Optional[datetime] = None) -> List[str]:
         return [partition.name for partition in self.partitions_def.get_partitions(current_time)]
 
+    def get_run_config_for_partition_key(self, key: str) -> Dict[str, Any]:
```

Comment:
It would be helpful to include docstring here. Also, I think we normally use `partition_key` for the name of partition key args.